server/pg/sql_collector.h

```diff
@@ -67,6 +69,12 @@ class Objects : public irs::memory::Managed {
   struct ObjectData {
     AccessType type = AccessType::None;
     std::shared_ptr<catalog::SchemaObject> object;
+
+    // TODO(mbkkt) Maybe remove this and instead make catalog::Table be able
+    // to implement connector::Table without allocation.
+    // This probably requires changing axiom::connector::Table.
+    void Ensure() const;
```

Comment:
NIT: Maybe `EnsureTable` to be more specific?

Reply:
ok, https://github.com/serenedb/serenedb/pull/71/changes/9130b384dc12cc112ab8c5c6745532a87ef55c55